src/Umbraco.Cms.Integrations.SEO.SemrushTools/Umbraco.Cms.Integrations.SEO.SemrushTools/Controllers/SemrushController.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+using Newtonsoft.Json;
+
+using Umbraco.Cms.Integrations.SEO.SemrushTools.Models.Dtos;
+using Umbraco.Cms.Integrations.SEO.SemrushTools.Services;
+using Umbraco.Web.Mvc;
+
+namespace Umbraco.Cms.Integrations.SEO.SemrushTools.Controllers
+{
+    [PluginController("UmbracoCmsIntegrationsSemrush")]
+    public class SemrushController : BaseController
+    {
+        private readonly HttpClient _client;
```

Comment:
You may want to make this static, rather than newing one up each time.  Have a read [here](https://www.aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/).  And you can compare with the pattern used in the HubSpot integration.  I'm not 100% sure here and there may be no issue with how you've done it, but maybe have a review of the other approach and see what you think.

Reply:
Item addressed.